.travis.yml

```diff
@@ -0,0 +1,26 @@
+# This config file for Travis CI utilizes ros-industrial/industrial_ci package.
+# For more info for the package, see https://github.com/ros-industrial/industrial_ci/blob/master/README.rst
+sudo: required 
+dist: trusty 
+language: generic 
+compiler:
+  - gcc
+notifications:
+  email:
+    on_success: always
+    on_failure: always
+    recipients:
+      - gm130s@gmail.com
```

Comment:
@robotpilot wrote:
> One question is that your email is in recipients, is this a requirement?

No it is not a required. This is just <s>boilerplate</s> me who's willing to watch the result so that when something's going wrong I may be able to assist asap. Your can of course remove it at your discretion.